py/autoevals/llm.py

```diff
@@ -68,11 +68,10 @@ def build_classification_functions(useCoT, choice_strings):
 class OpenAIScorer(Scorer):
     def __init__(
         self,
-        temperature=None,
         api_key=None,
         base_url=None,
     ):
-        self.extra_args = {"temperature": temperature or 0}
+        self.extra_args = {}
```

Comment:
@ankrgyl - Made a minor change here to move temperature from `OpenAIScorer` to LLM scorer, since moderation doesn't have a temperature.

Reply:
Ah. This wouldn't work - RAGAS implements `OpenAIScorer` - We need to differentiate between model-based scorer (Which supports temperature) vs non-model-based scorer (I guess `moderation` is the only one.). Update coming.

Reply:
OK - Added another abstraction `OpenAILLMScorer` that supports `temperature`: https://github.com/braintrustdata/autoevals/pull/63/files#diff-9f0a07ea81d0fec9ca54731e162add99bdcc55222f1bce86b2bf751311454721R81

Not sure if we've over-optimizing to support moderation endpoint. Lemme know.

Reply:
Ahh this seems like a clever fix. Thank you